src/pages/Main/MainCategory.js

```diff
@@ -0,0 +1,107 @@
+import React, { useState, useEffect } from 'react';
+import styled from 'styled-components';
+import variables from '../../styles/variables';
+import FilterBtn from '../List/FilterButton';
+
+const MainCategory = () => {
+  const [scrollY, setScrollY] = useState(0);
+  const isBtnActive = scrollY > 30;
+  const stickyCate = () => {
+    setScrollY(window.pageYOffset);
+  };
+
+  useEffect(() => {
+    window.addEventListener('scroll', stickyCate);
+    return () => {
+      window.removeEventListener('scroll', stickyCate);
+    };
+  }, []);
+
+  const [category, setCatecory] = useState([]);
+  useEffect(() => {
+    fetch('/data/mainMockData.json')
+      .then(res => res.json())
+      .then(data => {
+        const cateList = data.map(({ category }) => category);
+        const setCateList = cateList.filter((data, idx, arr) => {
+          return arr.findIndex(item => item.name === data.name) === idx;
+        });
+        setCatecory(setCateList);
+      });
+  }, []);
+
+  return (
+    <MainTop className={isBtnActive ? 'active' : ''}>
+      <CateWrap>
+        <CateItem>
+          {category.map(item => {
+            return (
+              <Cate key={item.id}>
+                <img src={item.img_url} alt={item.name} />
+                <span>{item.name}</span>
+              </Cate>
+            );
+          })}
```

Comment:
```suggestion
          {category.map(({id, img_url, name})=> {
            return (
              <Cate key={id}>
                <img src={img_url} alt={name} />
                <span>{name}</span>
              </Cate>
            );
          })}
```
여기서 바로 구조분해 할당 해주는 것도 깔끔해 보입니다!